apps/backoffice-v2/src/pages/Entity/components/Case/components/CaseOverview/CaseOverview.tsx

```diff
@@ -29,25 +29,29 @@ export const CaseOverview = ({ processes }: { processes: string[] }) => {
     workflow?.context?.pluginsOutput?.riskEvaluation?.riskIndicatorsByDomain ??
       workflow?.context?.pluginsOutput?.risk_evaluation?.riskIndicatorsByDomain ??
       {},
-  )?.map(([domain, riskIndicators]) => {
-    const tab = camelCase(domain.toLowerCase());
-    const isValidCaseTab = CaseTabs.includes(tab);
+  )
+    ?.map(([domain, riskIndicators]) => {
+      const domainTitle = domain ?? '';
+      const tabEntry = Object.entries(TabToLabel).find(([_, label]) => label === domainTitle);
+      const tab = tabEntry ? tabEntry[0] : camelCase(domainTitle.toLowerCase());
+      const isValidCaseTab = CaseTabs.includes(tab as any);
 
-    return {
-      title: TabToLabel[tab as keyof typeof TabToLabel] ?? titleCase(domain ?? ''),
-      search: isValidCaseTab
-        ? getUpdatedSearchParamsWithActiveTab({
-            tab: tab,
-          })
-        : undefined,
-      riskIndicators:
-        riskIndicators && Array.isArray(riskIndicators)
-          ? riskIndicators.map((riskIndicator: z.infer<typeof RiskIndicatorSchema>) => ({
-              name: riskIndicator.name,
-            }))
-          : [],
-    };
-  });
+      return {
+        title: domain,
```

Comment:
Should this be `domainTitle`?
```suggestion
        title: domainTitle,
```